src/main/webapp/app/modeling-assessment/modeling-assessment.component.scss

```diff
@@ -11,7 +11,6 @@ $draggable-width: 15px;
         align-items: center;
         justify-content: space-between;
         width: 100%;
-        z-index: 2;
```

Comment:
@jmaximilian2 Could not find any issues with removing the `z-index` of the blue header above the modelling editor on assessment mode. Please correct me, if I missed something, e.g. on conflict resolution screen.

As Apollon popovers are now rendered by appending them to the end of the `DOMTree`, they do not require a `z-index` as long as other elements do not have a `z-index` set.